packages/mural-client/src/index.ts

```diff
@@ -667,9 +706,17 @@ const buildApiClient = (
 
       return await response.json();
     },
-    searchCrossWorkspaceMurals: async ({ q }, options) => {
+    searchCrossWorkspaceMurals: async ({
+      q,
+      createdSince,
+      createdUntil,
+      createdFilterStrict,
+      updatedSince,
+      updatedUntil,
+      updatedFilterStrict }, options) => {
       const params = optionsParams(options);
-      const response = await fetchFn(api(`search/mural?q=${q}${params}`), {
+      const crossWorkspaceTimeParams = buildCrossWorkspaceParams(q, createdSince, createdUntil, createdFilterStrict, updatedSince, updatedUntil, updatedFilterStrict);
+      const response = await fetchFn(api(`search/mural?${crossWorkspaceTimeParams}?${params}`), {
```

Comment:
I think the second `?` should be removed

Reply:
Sounds good!